cargo-creusot/src/main.rs

```diff
@@ -10,11 +10,16 @@ mod helpers;
 use helpers::*;
 mod why3_launcher;
 use why3_launcher::*;
+mod why3find_wrapper;
+use why3find_wrapper::{why3find_config, why3find_prove};
+
 enum Subcommand {
     // subcommand to pass on to creusot-rustc
     Creusot(Option<CreusotSubCommand>),
-    // subcommand to handle in cargo-creusot
+    // subcommands to handle in cargo-creusot
     Setup(SetupSubCommand),
+    Config(ConfigArgs),
```

Comment:
Are there cases where it's useful for the user to manually call the `config` subcommand? Or could we instead not expose it and always rely on `prove`?

Reply:
Actually I misunderstood the purpose of `config` yesterday. The `why3find.json` that it generates is meant to be committed in git, so `config` should only be run when creating a project, and it shouldn't be run in `prove`.

I was confused because I was only thinking of the simplest projects which would just work with a basic config.

Reply:
OK, then having a separate config command makes sense for now! (Then ultimately when we make `creusot setup` project-local we should be able to combine `config` and `setup`)